framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Form/AbpSelectTagHelperService.cs

```diff
@@ -120,6 +120,8 @@ protected virtual void AddAutocompleteAttributes(TagHelperOutput output)
             output.Attributes.Add("data-autocomplete-filter-param-name", TagHelper.AutocompleteFilterParamName);
             output.Attributes.Add("data-autocomplete-selected-item-name", TagHelper.AutocompleteSelectedItemName);
             output.Attributes.Add("data-autocomplete-selected-item-value", TagHelper.AutocompleteSelectedItemValue);
+            output.Attributes.Add("data-allow-clear", TagHelper.AllowClear);
+            output.Attributes.Add("data-placeholder", TagHelper.Placeholder);
```

Comment:
They should have the same convention,

```suggestion
            output.Attributes.Add("data-autocomplete-allow-clear", TagHelper.AllowClear);
            output.Attributes.Add("data-autocomplete-placeholder", TagHelper.Placeholder);
```